tests/UI/campaigns/functional/BO/02_orders/01_orders/08_viewCustomer.js

```diff
@@ -0,0 +1,92 @@
+require('module-alias/register');
+
+// Helpers to open and close browser
+const helper = require('@utils/helpers');
+
+// Common tests login BO
+const loginCommon = require('@commonTests/loginBO');
+
+// Import pages
+const dashboardPage = require('@pages/BO/dashboard');
+const ordersPage = require('@pages/BO/orders');
+const customersPage = require('@pages/BO/customers/view');
+
+// Import customer 'J. DOE'
+const {DefaultAccount} = require('@data/demo/customer');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_orders_orders_viewCustomer';
+
+// Import expect from chai
+const {expect} = require('chai');
+
+
+// Browser and tab
+let browserContext;
+let page;
+
+/*
+Go to orders page
+Filter by customer name 'J. DOE'
+Click on customer link on grid
+Check that View customer page is displayed
+ */
+describe('View customer from orders page', async () => {
+  before(async function () {
+    browserContext = await helper.createBrowserContext(this.browser);
+    page = await helper.newTab(browserContext);
+  });
+
+  it('should login in BO', async function () {
+    await loginCommon.loginBO(this, page);
+  });
+
+  it('should go to orders page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToOrdersPage', baseContext);
+
+    await dashboardPage.goToSubMenu(
+      page,
+      dashboardPage.ordersParentLink,
+      dashboardPage.ordersLink,
+    );
+
+    await ordersPage.closeSfToolBar(page);
+
+    const pageTitle = await ordersPage.getPageTitle(page);
+    await expect(pageTitle).to.contains(ordersPage.pageTitle);
+  });
+
+  it('should reset all filters', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'resetFilters', baseContext);
+
+    const numberOfOrders = await ordersPage.resetAndGetNumberOfLines(page);
+    await expect(numberOfOrders).to.be.above(0);
+  });
+
+  it('should filter order by customer name', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'filterByCustomer', baseContext);
+
+    await ordersPage.filterOrders(
+      page,
+      'input',
+      'customer',
+      DefaultAccount.lastName,
+    );
+
+    const numberOfOrders = await ordersPage.getNumberOfElementInGrid(page);
+    await expect(numberOfOrders).to.be.at.least(1);
+  });
+
+  it('should check customer link', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'viewCustomer', baseContext);
+
+    // Click on customer link first row
+    page = await ordersPage.viewCustomer(page, 1);
+
+    const pageTitle = await customersPage.getPageTitle(page);
+    await expect(pageTitle).to
+      .contains(`${customersPage.pageTitle} ${DefaultAccount.firstName[0]}. ${DefaultAccount.lastName}`);
```

Comment:
Why is there a point after `${DefaultAccount.firstName[0]}` ? Is this normal ?

Reply:
Yes, it's normal. the goal here is ti display `J. DOE' instead of `John DOE`